papermill/execute.py

```diff
@@ -44,8 +44,8 @@ def _parameterize_notebook(notebook_path, output_path, parameters):
             val = '"%s"' % val  # TODO: Handle correctly escaping input strings.
         param_content += '%s = %s\n' % (var, val)
 
-    # Remove the first cell of the block and replace it with the param content.
-    nb = read_notebook(notebook_path)
+    # Find `parameters` cell and replaced with our parameters.
```

Comment:
I had two functions called read_notebook. One read a ipynb file and
returned a NotebookNode. The other is the API function that returns a
Papermill object. I removed the former to avoid confusion.